README.md

```diff
@@ -193,6 +193,8 @@ be removed in the future.
 
 -   `rust_decimal`: Add support for `NUMERIC` using the `rust_decimal` crate.
 
+-   `cube`: Add support for `cube`.
```

Comment:
This section of the README is for optional features but you didn't code it that way.

Which is perfectly fine, it doesn't need to be an optional feature unless it needs to bring in additional dependencies, but it's misleading to list it here.